### PR TITLE
removed outdated text

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8452

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: windows/security/threat-protection/security-policy-settings/password-policy.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ You can configure the password policy settings in the following location by usin
 
 **Computer Configuration\\Windows Settings\\Security Settings\\Account Policies\\Password Policy**
 
-If individual groups require distinct password policies, these groups should be separated into another domain or forest, based on additional requirements.
+This group policy is applied on domain level. If individual groups require distinct password policies, consider using of fine-grained password policies, as described above.
```
Comment (illfated) [inline windows/security/threat-protection/security-policy-settings/password-policy.md:41]: Suggestion:
```suggestion
This group policy is applied on the domain level. If individual groups require distinct password policies, consider the use of fine-grained password policies, as described above.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - only @illfated's suggestions.

**Review by denisebmsft — APPROVED** [review]
Thank you @MaratMussabekov

**Pull request merged**